src/main/java/org/icgc/argo/program_service/controller/ProgramController.java

```diff
@@ -71,10 +71,15 @@ public ResponseEntity<UpdateProgramResponseDTO> updateProgram(
           final String authorization,
       @RequestBody UpdateProgramRequestDTO updateProgramRequestDTO) {
     authorizationService.requireDCCAdmin(authorization);
+    UpdateProgramRequest request;
     UpdateProgramResponse response;
     try {
-      response = serviceFacade.updateProgramWithDataCenter(updateProgramRequestDTO);
-    } catch (NotFoundException | NoSuchElementException e) {
+      request =
+          grpc2JsonConverter.fromJson(
+              grpc2JsonConverter.getJsonFromObject(updateProgramRequestDTO),
+              UpdateProgramRequest.class);
+      response = serviceFacade.updateProgram(request);
+    } catch (NotFoundException | NoSuchElementException | IOException e) {

```

Comment:
Accidentally included these changes which revert updates we made in #437 . Lets remove these changes